chained/proxy.go

```diff
@@ -212,9 +214,13 @@ func createImpl(configDir, name, addr, transport string, s *config.ProxyConfig,
 	case "algeneva":
 		impl, err = newAlgenevaImpl(addr, s, reportDialCore)
 	case "water":
-		impl, err = newWaterImpl(addr, s, reportDialCore)
+		waterDir := filepath.Join(configDir, "water")
+		if err := os.MkdirAll(waterDir, 0755); err != nil {
```

Comment:
This will fail when the directory already exists.  Explanation [here](https://github.com/getlantern/flashlight/pull/1427#discussion_r1817239347).

Reply:
Sorry, ignore me, I was mixed up.  This looks good!